apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -2053,7 +2053,7 @@
       "SEARCH_INFO": "You can search users by name, email or login or you can list all users without vo.",
       "NO_USERS_FOUND": "No users found.",
       "EMPTY_SEARCH": "Search field cannot be empty",
-      "USERS_WITHOUT_VO": "Find users without organization"
+      "USERS_WITHOUT_VO": "Search in users without organization"
     },
```

Comment:
Personaly I would prefer either "Find users ..." or "Search for users ...". "Search in users ..." feels unnatural.